docs/LwM2M_Client/Nordic/nRF7002.md

```diff
@@ -6,7 +6,7 @@ Integrate your Nordic nRF7002 DK board.
 
 - The nRF7002 DK board with a USB cable.
 - Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
```

Comment:
So if we create more macros I suggest creating macro with this line too

Reply:
Well, I should have done it in the previous PR